app/backend/src/couchers/crypto.py

```diff
@@ -149,7 +148,7 @@ def aead_generate_key():
     return random_bytes(_aead_key_len)
 
 
-def aead_encrypt(key: bytes, secret_data: bytes, plaintext_data: bytes = b"", nonce: Optional[bytes] = None) -> bytes:
+def aead_encrypt(key: bytes, secret_data: bytes, plaintext_data: bytes = b"", nonce: bytes | None = None):
```

Comment:
Seems you accidentally dropped the output type

Reply:
@aapeliv I didn't! Pushed it now.